pkg/auth/is_jwt_token_expired.go

```diff
@@ -0,0 +1,17 @@
+package auth
+
+import (
+	"time"
+
+	"github.com/golang-jwt/jwt/v4"
+)
+
+// This method returns true if JWT token is expired, otherwise returns false
+func IsJWTTokenExpired(accessToken string) bool {
+	if token, _ := jwt.Parse(accessToken, nil); token != nil {
+		tokenClaims := token.Claims.(jwt.MapClaims)
+		exp := tokenClaims["exp"].(float64)
+		return exp-float64(time.Now().Unix()) <= 0
```

Comment:
I think 	"github.com/golang-jwt/jwt/v4" itself provides you with that information.

See https://pkg.go.dev/github.com/golang-jwt/jwt/v4@v4.2.0#pkg-constants: `The errors that might occur when parsing and validating a token `.
In that case you can check for the `jwt.ValidationErrorExpired` error and maybe `|jwt.ValidationErrorNotValidYet` (depending on what we understand by "Is JWT token expired" method and where it is used. You can see an example of the error handling in the examples in https://pkg.go.dev/github.com/golang-jwt/jwt/v4@v4.2.0#Parse